webpack.config.js

```diff
@@ -19,7 +19,7 @@ var config = {
             loader: 'babel-loader',
 
             query: {
-               presets: ['es2015', 'react']
+               presets: ['env', 'react']
```

Comment:
je ne vois pas pourquoi "env", je laisserai es2015 qui permet de dire à Babel de transpiler le es2015 en es5

Reply:
dis moi pk env ?

Reply:
https://babeljs.io/docs/plugins/preset-env/

c'est la nouvelle nomenclature

Without any configuration options, babel-preset-env behaves exactly the same as babel-preset-latest (or babel-preset-es2015, babel-preset-es2016, and babel-preset-es2017 together).

Reply:
sinon quoi de neuf depuis l'temps ta famille va bien ? marco grandit ca lui fait combien maintenant 3ans le 8 février prochain

Reply:
Ecoute ça va bien et toi ? ta famille ? t'as fait un p'tit frère ou p'tite soeur à ta fille ?
oui 3 ans le 8 février prochain, ça grandit..
Tu te rappel de la ddn de mon fils mais la mienne le 8 janv, kedal :)

Reply:
ben tu te souviens pas que ma copine a accoucher quand j'était encore a eliberty !!!! jsuis sure qu'il y a encore le faire part accrocher au frigo ....

sinon ben toi j avoue je lai jamais su mais ton fils est née le mm jr que ma fille donc vi je men souvient.

sinon comment va l'équipe pas trop dure cette saison ?

Reply:
ah mais oui ! Logan.

Ou la si on en a bien bavé... Npy a complètement migré sur le nouveau système.. maintenant il me reste encore 2 ou 3 ppu à gérer et on sera pas mal.

Bon et toi le taf ça se passe

Reply:
on fait aller comme dab, j'ai pue avoir une semaine de vac entre noël et le nouvelle ans 
ca c'était cool 
et toi ca a changer de ton coté ? a part thomas qui est partie l'equipe est tjr ho complete patpat na pas desimer tout le monde :-))